packages/xo-server-sdn-controller/src/ovsdb-client.js

```diff
@@ -112,41 +122,43 @@ export class OvsdbClient {
     const portName = bridgeName + '_port' + index
 
     // Add interface and port to the bridge
-    const options = ['map', [['remote_ip', remoteAddress], ['key', key]]]
+    const options = { remote_ip: remoteAddress, key: key }
     if (password !== undefined) {
-      options[1].push(['psk', password])
+      options.psk = password
     }
-
-    const otherConfig =
-      remoteNetwork !== undefined
-        ? ['map', [['xo:sdn-controller:cross-pool', remoteNetwork]]]
-        : ['map', [['xo:sdn-controller:private-pool-wide', 'true']]]
-
     const addInterfaceOperation = {
       op: 'insert',
       table: 'Interface',
       row: {
         type: encapsulation,
-        options: options,
+        options: toOvsdbMap(options),
         name: interfaceName,
       },
       'uuid-name': 'new_iface',
     }
+
+    const otherConfig =
+      remoteNetwork !== undefined
+        ? { 'xo:sdn-controller:cross-pool': remoteNetwork }
+        : { 'xo:sdn-controller:private-pool-wide': 'true' }
```

Comment:
Why did you move this code?

Reply:
otherConfig used to be used twice (but one was useless so removed), now only once so I put the declaration just above the only use.

Reply:
Inline it then.